README.md

```diff
@@ -1,12 +1,109 @@
+
+## Project
+
+This Peer Assessment Application is designed to streamline the process of evaluating teamwork in university group projects. The application allows students to anonymously assess their peers across four key performance dimensions: cooperation, conceptual contribution, practical contribution, and work ethic. By providing structured feedback, the system encourages accountability and helps improve team collaboration.
+
+Instructors can easily manage teams and track student progress through a comprehensive dashboard that displays aggregated scores and detailed assessments. The system also supports CSV export of the evaluation results, making it easier for instructors to analyze performance and provide guidance. The application is built with scalability in mind, ensuring it can be adapted for different academic environments and expanded with new features in future iterations.
+
+## Installation Process
+
+## Pre-requisites
+
+Ensure you have the following software installed on your machine:
+
+- Git: For version control and collaboration.
+- NVM (Node Version Manager): To manage Node.js versions.
+- Go: For backend development.
+
+## Steps
+
+#### 1. Clone the Repository
+
+First, clone the project repository from GitHub to your local machine :
+
+```bash
+git clone https://github.com/<team_name>/<repo_name>.git
+cd <repo_name>
+```
+
+#### 2. Frontend Setup
+
+- Navigate to the frontend directory :
+
+```bash
+cd frontend/
+```
+
+- Install all required frontend libraries and dependencies :
+
+```bash
+npm install
+```
+
+- To build the production ready version of the frontend, run the code which will generate the dist/ folder containing the optimized frontend assets :
+
+```bash
+npm run build
+```
+
+- To host the frontend locally by running which will start a local development server, and the application will be accessible :
+
+```bash
+npm run dev
+
+http://localhost:8080
+```
+
+#### 3. Backend Setup
+
+- Navigate to the backend directory :
+
+```bash
+cd backend/
+```
+
+- To host the backend server :
+
+```bash
+go run main.go
+```
+
 ## Collaborators
-| Name | Concordia ID |
-| -------------- | --- |
-| Anh Thien Nguyen | Upon request |
-| Daniel Lam | 40248073 |
-| Juan-Carlos Sreng-Flores | 40101813 |
-| Ming-Yang Calvin Lee | 40264581 |
-| Mouhamed Coundoul | 40248237 |
-| Samuditha Wijenarayana| 40224895 |
+
+| Name                     | Concordia ID |
+| ------------------------ | ------------ |
+| Anh Thien Nguyen         | Upon request |
+| Daniel Lam               | 40248073     |
+| Juan-Carlos Sreng-Flores | 40101813     |
+| Ming-Yang Calvin Lee     | 40264581     |
+| Mouhamed Coundoul        | 40248237     |
+| Samuditha Wijenarayana   | 40224895     |
+
+## Technologies
+
+⚫ Frontend: HTML, CSS, JavaScript, React
```

Comment:
Should we mention Vite here?
Also should it be JavsScript or TypeScript?

Reply:
Yeah I guess we can change `React` for `Vite React` and `JavaScript` for `TypeScript`